pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -793,12 +911,21 @@ class CreateVmModal extends React.Component {
 
                 <hr />
 
-                <label className="checkbox-inline">
-                    <input id="start-vm" type="checkbox"
-                        checked={this.state.startVm}
-                        onChange={e => this.onValueChanged('startVm', e.target.checked)} />
-                    {_("Immediately Start VM")}
-                </label>
+                {this.state.sourceType != PXE_SOURCE &&
+                 this.state.sourceType != EXISTING_DISK_IMAGE_SOURCE &&
+                 this.props.unattendedSupported &&
```

Comment:
Why does unattended installation not work with PXE or disk images? Isn't that just a function of whether the installation runs kickstart? That sounds rather independent of whether the bits are delivered from an iso, from an image file, or through the network?

Reply:
With disk images you have no installation phase to perform.
Investigating the PXE scenario...

Reply:
So I communicated with the libvirt devels, the answer was the following:
```
It's not possible to setup unattended installation if you don't modify the PXE server setting which virt-install can't do
```
So I guess --unattended mode does not work with --pxe yet.

Reply:
Ack, thanks for checking!

Reply:
Hmm, but now you changed the PR to drop the `this.state.sourceType != PXE_SOURCE` check, I thought it was correct after all?

Reply:
Ups. yes I removed it for testing if it's feasible to use it and by mistake I didn't put it back.
Thanks.